packages/core/src/Param/Param.ts

```diff
@@ -72,6 +77,39 @@ export type ParamValue = Param['value']
 
 // // quick param builders
 
+type StringableConfigType = Omit<StringableParam, 'value' | 'type'> & {
+  value: StringableInputValue['value']
+}
+
+export const createDefaultStringable = ({
```

Comment:
Is there a reason for introducing this over using str()? I see this is the one now used in the Computers param definition.
Purpose of str() was to allow for simple creation by giving sensible defaults (include all evaluations, casts by default; use name as lable, etc), but still have it output the full secure type. So I think we should prefer to configure str() with "batteries included"/all options available and then use that in Computers.

Reply:
I decided against using `str` for a couple of reasons:
1. The default `str` function doesn't support passing in `casts` and `interpolationsFromPort` params.
2. `str` is already configured with `casts: [stringCast]` by default.